### PR TITLE
Add function to instrument to validate all gettable parameters

### PR DESCRIPTION
**Pull request by peendebak**
This is useful when connecting to an instrument for the first time to check that all values are in the required range.

@giulioungaretti @alexcjohnson

### CODE REVIEW

File: qcodes/tests/test_instrument.py [hunk 1]
```diff
@@ -53,7 +53,8 @@ def setUpClass(cls):
 
         cls.gates = MockGates(model=cls.model, server_name='')
         cls.source = MockSource(model=cls.model, server_name='')
-        cls.meter = MockMeter(model=cls.model, keep_history=False, server_name='')
+        cls.meter = MockMeter(
+            model=cls.model, keep_history=False, server_name='')
```
Comment (peendebak) [inline qcodes/tests/test_instrument.py]: My autopep8 formatter says its fine

File: qcodes/instrument/base.py [hunk 2]
```diff
@@ -628,3 +628,12 @@ def __getstate__(self):
             'were trying to use a local instrument (defined with '
             'server_name=None) in a background Loop. Local instruments can '
             'only be used in Loops with background=False.')
+
+    def validate_status(self, verbose=0):
+        """ Validate the values of all gettable parameters """
```
Comment (peendebak) [inline qcodes/instrument/base.py]: The `verbose` parameter is an int on purpose. In general I use the convetion: 0: no output, 1: normal output, >=2: used for debugging, output not stable

File: qcodes/tests/test_instrument.py (hunk 1 above)
Comment (giulioungaretti) [inline qcodes/tests/test_instrument.py]: https://www.python.org/dev/peps/pep-0008/#indentation

Comment (giulioungaretti) [inline qcodes/tests/test_instrument.py]: cool

File: qcodes/instrument/base.py (hunk 2 above)
Comment (giulioungaretti) [inline qcodes/instrument/base.py]: a) Cool,  just write the assumption in the docstring ! 

b) In this function 2 and 1 are the same then, which I don't like, and I don't see it documented or understandable from the code. 
Change to true and false in this  case.

**Pull request closed without merging**